### PR TITLE
Multicluster version routing blog

### PR DESCRIPTION
**Pull request by frankbu**

### CODE REVIEW

File: content/blog/2019/multicluster-version-routing/index.md [hunk 1]
```diff
@@ -0,0 +1,455 @@
...
+  name: reviews-global
+spec:
+  host: reviews.default.global
+  trafficPolicy:
+    tls:
+      mode: ISTIO_MUTUAL
+  subsets:
+  - name: v1
+    labels:
+      version: v1
```
Comment (rshriram) [inline content/blog/2019/multicluster-version-routing/index.md]: you dont need v1 here right?

Comment (frankbu) [reply content/blog/2019/multicluster-version-routing/index.md]: True, but I figured it was harmless.
I'll remove it.

File: content/blog/2019/multicluster-version-routing/index.md [hunk 2]
```diff
@@ -0,0 +1,458 @@
...
+## Create a virtual service to route reviews service traffic
+
+At this point, all calls to the `reviews` service will go to the local `reviews` pods (`v1`) because
+if you look at the source code you will see that the `productpage` implementation is simply making
+requests to `http://reviews:9080` (which expands to host `reviews.default.svc.cluster.local`), the
+local version of the service.
+The corresponding remote service is named `reviews.default.global`, so route rules are needed to
+redirect requests to the global host.
+
+> Note that if all of the versions of the `reviews` service were remote, so there is no local `reviews`
```
Comment (geeknoid) [inline content/blog/2019/multicluster-version-routing/index.md]: Instead of this, please use the new {{< warning | tip | idea >}} syntax.

Comment (geeknoid) [reply content/blog/2019/multicluster-version-routing/index.md]: Here and elsewhere

File: content/docs/setup/kubernetes/multicluster-install/gateways/index.md [hunk 3]
```diff
@@ -161,8 +164,10 @@ Every service in a given cluster that needs to be accessed from a different remo
 cluster requires a `ServiceEntry` configuration in the remote cluster.
 The host used in the service entry should be of the form `<name>.<namespace>.global`
 where name and namespace correspond to the service's name and namespace respectively.
-Visit our [multicluster using gateways](/docs/examples/multicluster/gateways/)
-example for detailed configuration instructions.
+
+To confirm that your multicluster configuration is working, we suggest proceeding to our
```
Comment (geeknoid) [inline content/docs/setup/kubernetes/multicluster-install/gateways/index.md]: proceeding -> you proceed

**Pull request merged**